apim-with-analytics/apim-analytics-dashboard/config/conf/dashboard/deployment.yaml

```diff
@@ -0,0 +1,386 @@
+          connectionTestQuery: SELECT 1
+          validationTimeout: 30000
+          isAutoCommit: false
+
+    #Data source for APIM Analytics
+    - name: APIM_ANALYTICS_DB
+      description: Datasource used for APIM Analytics
+      jndiConfig:
+        name: jdbc/APIM_ANALYTICS_DB
+      definition:
+        type: RDBMS
+        configuration:
+          jdbcUrl: 'jdbc:mysql://172.28.128.3:3306/WSO2AM_STATS_DB?useSSL=false'
+          username: wso2carbon
+          password: wso2carbon
+          driverClassName: com.mysql.cj.jdbc.Driver
+          maxPoolSize: 50
+          idleTimeout: 60000
+          connectionTestQuery: SELECT 1
+          validationTimeout: 30000
+          isAutoCommit: false
+
+    #   # Sample datasource used in API Manager
+    - name: AM_DB
+      description: Main datasource used by API Manager
+      jndiConfig:
+        name: jdbc/AM_DB
+      definition:
+        type: RDBMS
+        configuration:
+          jdbcUrl: 'jdbc:mysql://172.28.128.3:3306/WSO2AM_DB?useSSL=false'
+          username: wso2carbon
+          password: wso2carbon
+          driverClassName: com.mysql.cj.jdbc.Driver
+          maxPoolSize: 10
+          idleTimeout: 60000
+          connectionTestQuery: SELECT 1
+          validationTimeout: 30000
+          isAutoCommit: false
+
+wso2.business.rules.manager:
+  datasource: BUSINESS_RULES_DB
+  # rule template wise configuration for deploying business rules
+  deployment_configs:
+    -
+      # <IP>:<HTTPS Port> of the Worker node
+      localhost:9444:
+        # UUIDs of rule templates that are needed to be deployed on the node
+        - stock-data-analysis
+        - stock-exchange-input
+        - stock-exchange-output
+        - identifying-continuous-production-decrease
+        - popular-tweets-analysis
+        - http-analytics-processing
+        - message-tracing-source-template
+        - message-tracing-app-template
+  # credentials for worker nodes
+  username: admin
+  password: admin
+
+wso2.transport.http:
+  transportProperties:
+    - name: "server.bootstrap.socket.timeout"
+      value: 60
+    - name: "client.bootstrap.socket.timeout"
+      value: 60
+    - name: "latency.metrics.enabled"
+      value: true
+
+  listenerConfigurations:
+    - id: "default-https"
+      host: "0.0.0.0"
+      port: 9643
+      scheme: https
+      keyStoreFile: "${carbon.home}/resources/security/wso2carbon.jks"
+      keyStorePassword: wso2carbon
+      certPass: wso2carbon
+
+# Authentication configuration
+auth.configs:
+  type: apim
+  ssoEnabled: true
+  properties:
+    adminScope: apim_analytics:admin
+    allScopes: apim_analytics:admin apim_analytics:creator apim_analytics:publisher_owner apim_analytics:publisher_editor apim_analytics:publisher_viewer apim_analytics:subscriber_owner apim_analytics:subscriber_editor apim_analytics:subscriber_viewer apim_analytics:everyone openid apim:api_view apim:subscribe
+    adminServiceBaseUrl: https://api-manager:9443
```

Comment:
@RidmiR I think this hostname `api-manager` is not applicable in the case of Vagrant. We have to refer the API Manager service using the API Manager VM IP.

Reply:
Done. Please check the latest commit.